src/lib/Scenes/Partner/Components/PartnerShows.tsx

```diff
@@ -55,62 +54,90 @@ export const PartnerShows: React.FC<{
   partner: PartnerShows_partner
   relay: RelayPaginationProp
 }> = ({ partner, relay }) => {
-  const [hasRecentShows, setHasRecentShows] = useState(false)
-  const [fetchingNextPage, setFetchingNextPage] = useState(false)
-
-  const { scrollOffsetY, contentHeight, layoutHeight } = useStickyTabContext()
-
-  useOnCloseToBottom({
-    contentHeight,
-    scrollOffsetY,
-    layoutHeight,
-    callback: () => {
-      if (fetchingNextPage || !relay.hasMore()) {
-        return
+  const [isLoadingMore, setIsLoadingMore] = useState(false)
+
+  const hasRecentShows = partner.recentShows.edges.length > 0
+
+  const pastShows = partner.pastShows && partner.pastShows.edges
+
+  const sections: StickyTabSection[] = useMemo(
+    () => {
+      if (!pastShows && !hasRecentShows) {
+        return [
+          {
+            key: "empty_state",
+            content: <TabEmptyState text="There are no shows from this gallery yet" />,
+          },
+        ]
```

Comment:
`FlatList` actually has an `EmptyStateComponent` prop so we could use that and return an empty array here instead. no biggie though